cirq-core/cirq/ops/pauli_string.py

```diff
@@ -1371,7 +1370,43 @@ def inplace_before(self, ops: cirq.OP_TREE) -> cirq.MutablePauliString:
         Returns:
             The mutable pauli string that was mutated.
         """
-        return self.inplace_after(protocols.inverse(ops))
+        # An inplace impl of PauliString.conjugated_by().
+        flattened_ops = list(op_tree.flatten_to_ops(ops))
+
+        for op in flattened_ops[::-1]:
+            conjugated: cirq.DensePauliString = dense_pauli_string.DensePauliString(
```

Comment:
`conjugates = self.dense(self.qubits) * dense_pauli_string.DensePauliString('I'*len(self.qubits))` should do the trick ... no need for the logic below

Reply:
Good idea! updated.